src/qt/fontenum.cpp

```diff
@@ -7,13 +7,23 @@
 
 // For compilers that support precompilation, includes "wx.h".
 #include "wx/wxprec.h"
-
 #include "wx/fontenum.h"
+#include <QFontDatabase>
 
-bool wxFontEnumerator::EnumerateFacenames( wxFontEncoding WXUNUSED(encoding),
-                bool WXUNUSED(fixedWidthOnly))
+bool wxFontEnumerator::EnumerateFacenames( wxFontEncoding WXUNUSED(encoding), bool fixedWidthOnly)
```

Comment:
`encoding` will probably need a mapping between `wxFontEncoding` and `QFontDatabase::WritingSystem`.

```suggestion
bool wxFontEnumerator::EnumerateFacenames(wxFontEncoding WXUNUSED(encoding), bool fixedWidthOnly)
```

Reply:
I'll apply these changes when I fix up the include paths and build error